gitbook-docs/market/user-guides/sign-up.md

```diff
@@ -1,24 +1,30 @@
 # Sign up
 
-Access to the market requires authentication. Please ensure you have a valid username/password. If you have not registered, please look at the instructions on how to do so [here](../front-end.md#new-user-signup) via the front end.
+Access to the market requires authentication.
+Make sure you have a valid username and password.
+If you have not registered, please look at the instructions on how to do so
+[via the web interface](../front-end.md#new-user-signup).
 
-Or, you can use the below command line in the `proof-market-toolchain` repository.
+Or, you can use the following command in the `proof-market-toolchain` repository.
 
 ## User
 
-Users can submit/retrieve orders on the proof market. You cannot generate proofs if you sign up as an ordinary user.
+Signup for an ordinary user through CLI looks like this:
 
-```
+```bash
 python3 scripts/signup.py user -u <username> -p <password> -e <e-mail>
 ```
 
-## Proof Producer
+All users can submit and retrieve orders on Proof Market, but only proof producers
+can generate and submit proofs.
 
-Proof Producers can submit/retrieve orders on the proof market and have the additional ability to submit/generate proofs.&#x20;
+## Proof producer
 
-```
+Signup for a proof producer goes as follows:
+
+```bash
 python3 scripts/signup.py producer -u <username> -p <password> -e <e-mail>
```

Comment:
There are more input parameters for producers
https://github.com/NilFoundation/proof-market-toolchain/blob/master/scripts/signup.py#L68-L83

Reply:
And as far as I remember, one should start by making a "user" account and then proceed by registering this account as a producer. I may be wrong, though. Ask Ilia or Gurgen.

Reply:
I did, waiting for an answer

Reply:
You're right, I've updated this part. We should also update the CLI guides then